exercises/change/change_test.rb

```diff
@@ -45,16 +45,22 @@ def test_no_coins_make_0_change
 
   def test_error_testing_for_change_smaller_than_the_smallest_of_coins
     skip
-    assert_equal -1, Change.generate([5, 10], 3)
+    assert_raises(ArgumentError) do
+      Change.generate([5, 10], 3)
+    end
   end
 
   def test_error_if_no_combination_can_add_up_to_target
     skip
-    assert_equal -1, Change.generate([5, 10], 94)
+    assert_raises(ArgumentError) do
+      Change.generate([5, 10], 94)
+    end
   end
 
   def test_cannot_find_negative_change_values
     skip
-    assert_equal -1, Change.generate([1, 2, 5], -5)
+    assert_raises(ArgumentError) do
+      Change.generate([1, 2, 5], -5)
```

Comment:
`ArgumentError` is reasonable here.

(I don't think invalid input like this is something we should be testing for, but that's an unrelated issue. (and very much a matter of opinion.))